service/service_suite_test.go

```diff
@@ -126,4 +125,3 @@ func TestService(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecsWithDefaultAndCustomReporters(t, "P-Redis Smoke Tests", testReporter)
```

Comment:
This method is deprecated, right? It would be good if we create on our board a new story just for all of the ginkgo v2 deprecations that are not addressed.

Reply:
Yes, will create story to track it.